### DISCUSSION TITLE
How to change layout from A|B|C to A|(B/C)

### DISCUSSION THREAD

**Discussion by mat-m** [i3/i3#5542] [Q&A]
Hello

I just started to tackle with i3wm this week-end. Pretty easy so far, as I am a keyboard guy. 
But I have a use case I can't sort out.
I opened three terminals which went `|A|B|C|` with normal layout.

I would like to have
```
|A|B|
|A|-|
|A|C|
```
without killing C.

Trials:
* If I kill C, I know I can focus B, Mod+V, Mod+enter, and done.
* If I select C and apply Mod+Shift+K I will get
```
|A|B|
|---|
| C |
```
* I tried to select parent with Mod+A, then Mod+V, focus C, Mod+Shift+K, but same issue.

I don't know how to do it with an existing window.

Thank you for guiding me. 

Thank you

**Comment by ntcarlson** [comment]
There are lots of different ways you can do this. It might be helpful to think about i3's underlying window tree structure. Initially, the tree looks like `H[A B C]` and you want it to be `H[A V[B C]]`.

You can:

* Focus `B` and vertically split it with `split v` (default `$mod+v`). 
Tree structure is now `H[A V[B] C]`.
Next, focus `C` and move it left into the vertical split with `move left` (default `$mod+j`).
Tree structure is now `H[A V[B C]]`.

* Switch layout to a vertical split with `layout toggle split` (default `$mod+e`).
Tree structure is now `V[A B C]`.
Next, focus `A` and move it left out of the vertical split.
Tree structure is now `H[A V[B C]]`.

* Use the mouse (see [`tiling_drag`](https://i3wm.org/docs/userguide.html#tiling_drag)). While holding `$mod`, drag window `C` to the bottom of `B`.

**Reply by mat-m** [reply]
Writing the structure helps !

Thanks for this
